geoips/plugins/yaml/products/abi.yaml

```diff
@@ -1,28 +1,64 @@
-interface: product_inputs
-metadata:
-  name: abi
-  family: standard
-  short-description: "The abi product_inputs configuration"
-  long-description:
-    "The abi product_inputs configuration, which produces Infrared products, Vertical Velocity products, and the Visible IR products"
+interface: products
+family: list
+name: abi
+docstring: |
+  The abi products configuration, which produces Infrared products, Vertical Velocity products, and the Visible IR products
 spec:
   products:
     - name: Infrared
-      variables: ['B14BT']
+      docstring: |
```

Comment:
Do these all have to have docstring ?  If so,  are they ABI product definitions,  rather than product defaults ?  I don't really care if this gets changed or not,  just double checking terminology,  etc.

Reply:
In the case where we are using `product_defaults` we can optionally supply `docstring`. If `docstring` is provided then it is used. If it is not provided it is collected from the `product_defaults` plugin.

Does that seem appropriate?

Reply:
Slack conversation:
Jeremy
  [5:35 PM](https://geoips.slack.com/archives/D3R03CK1S/p1682724948976369)
It tests. The only thing unresolved is the docstring question.
In the case where we are using product_defaults we can optionally supply docstring. If docstring is provided then it is used. If it is not provided it is collected from the product_defaults plugin.
Does that seem appropriate?
[5:36](https://geoips.slack.com/archives/D3R03CK1S/p1682724967504659)
If we’re using family, docstring is required.
mindy
  [5:36 PM](https://geoips.slack.com/archives/D3R03CK1S/p1682724970032489)
Yes that is appropriate
Jeremy
  [5:36 PM](https://geoips.slack.com/archives/D3R03CK1S/p1682724978576059)
Great, I’ll resolve, test, and merge then.